shamuna/van.css

```diff
@@ -38,9 +40,11 @@ a{
 .small-container{
     display:flex;
     justify-content: space-between;
-    width:10000px;
+    width:51822px;
```

Comment:
Why such a huge number?) I think it's a pretty bad idea to hardcode width in px for this task, I think you could even avoid setting up width, that's how you could do it:
- Leave just one container instead of two
- container should have overflow-x: auto or scroll,  width: auto, justify-content: flex-start
- Each card should have width and height in px and some margin